stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -380,7 +387,7 @@ internal class ChatDomainImpl internal constructor(
         return result
     }
 
-    suspend fun createChannel(c: Channel): Result<Channel> =
+    suspend fun createNewChannel(c: Channel): Result<Channel> =
```

Comment:
This is an internal function. I needed to change its name because there would be a conflict with the use case function name.

Reply:
It is only used internally in offline SDK, right?
Can you modify the visibility of this method?

Reply:
Yep, added `internal` modifier